python/cudf/cudf/tests/test_orc.py

```diff
@@ -202,13 +202,21 @@ def test_orc_read_stripe(datadir, engine):
 
     num_rows, stripes, col_names = cudf.io.read_orc_metadata(path)
 
+    # Read stripes one at a time
     gdf = [
-        cudf.read_orc(path, engine=engine, stripe=i) for i in range(stripes)
+        cudf.read_orc(path, engine=engine, stripes=[i]) for i in range(stripes)
     ]
     gdf = cudf.concat(gdf).reset_index(drop=True)
+    assert_eq(pdf, gdf, check_categorical=False)
 
+    # Read stripes all at once
+    gdf = cudf.read_orc(path, engine=engine, stripes=range(stripes))
     assert_eq(pdf, gdf, check_categorical=False)
 
+    # Read only some stripes
+    gdf = cudf.read_orc(path, engine=engine, stripes=[0, 1])
```

Comment:
Can we test reading some non-contiguous stripes as well? I.E. `stripes=[1, 3]`?

Reply:
I added a test for this case and updated the Python documentation comment.